### ISSUE TITLE
Duffle driver documentation - prepending 'duffle-'

### ISSUE THREAD

**Issue by BenWhiting** [cnabio/duffle#889]
There might be a different issue out there but wanted to call this out for record sake.

https://github.com/cnabio/duffle/blob/master/docs/proposal/202-drivers.md#custom-drivers
"prepending duffle- to the driver name"

Duffle uses the cnab-go lib for this, which actually applies "cnab-"

https://github.com/cnabio/cnab-go/blob/af30b09b45da1b93ca09bc54349d93e661786b6d/driver/command/command.go#L45

**Comment by carolynvs** [comment]
I agree that the docs were originally correct when the command driver lived inside the duffle codebase and when we moved it, should have been updated to reflect the new behavior. Thanks for calling that out!

If anyone wants a small good first issue, the change needed is to update the documentation that Ben found to describe the new behavior, that duffle expects a "cnab-" prefix instead of "duffle-".
